addons/pandora/model/entity.gd

```diff
@@ -1,5 +1,6 @@
 ## An entity acts a container for properties and is used to represent
 ## a category or an actual concept in any game.
+@tool
```

Comment:
It is critical that `PandoraEntity` (and all its custom scripts) are a tool script, as this needs to be accessible inside the editor. No clue if there is another way to solve this!